driver.go

```diff
@@ -153,7 +153,11 @@ func (c ocConn) Ping(ctx context.Context) (err error) {
 
 	if c.options.Ping && (c.options.AllowRoot || trace.FromContext(ctx) != nil) {
 		var span *trace.Span
-		ctx, span = trace.StartSpan(ctx, "sql:ping",
+		spanName := "sql.ping"
+		if c.options.FormatSpanName != nil && c.options.FormatSpanName(ctx) != "" {
```

Comment:
Maybe this should be pulled out into a function and expanded a bit (to check for empty span name)

```golang
func (c ocConn) getSpanName(ctx context.Context, defaultName string) (spanName string) {
	if c.options.FormatSpanName != nil && c.options.FormatSpanName(ctx) != "" {
		spanName = c.options.FormatSpanName(ctx)
	}

	if spanName == "" {
		spanName = defaultName
	}

	return spanName
}
```